doc/developers/advanced_installation.rst

```diff
@@ -191,132 +267,72 @@ Then you need to set the following environment variables::
     export CXXFLAGS="$CXXFLAGS -I/usr/local/opt/libomp/include"
     export LDFLAGS="$LDFLAGS -Wl,-rpath,/usr/local/opt/libomp/lib -L/usr/local/opt/libomp/lib -lomp"
 
-Finally, build scikit-learn in verbose mode::
+Finally, build scikit-learn in verbose mode (to check for the presence of the
+``-fopenmp`` flag in the compiler commands)::
 
+    make clean
     pip install --verbose --editable .
 
-FreeBSD
--------
-
-The clang compiler included in FreeBSD 12.0 and 11.2 base systems does not 
-include OpenMP support. You need to install the `openmp` library from packages 
-(or ports)::
-
-    sudo pkg install openmp
-    
-This will install header files in ``/usr/local/include`` and libs in 
-``/usr/local/lib``. Since these directories are not searched by default, you 
-can set the environment variables to these locations::
-
-    export CFLAGS="$CFLAGS -I/usr/local/include"
-    export CXXFLAGS="$CXXFLAGS -I/usr/local/include"
-    export LDFLAGS="$LDFLAGS -Wl,-rpath,/usr/local/lib -L/usr/local/lib -lomp"
-
-Finally you can build the package using the standard command.
-
-For the upcomming FreeBSD 12.1 and 11.3 versions, OpenMP will be included in 
-the base system and these steps will not be necessary.
-
-
-Installing build dependencies
-=============================
-
 Linux
 -----
 
-Installing from source without conda requires you to have installed the
-scikit-learn runtime dependencies, Python development headers and a working
-C/C++ compiler. Under Debian-based operating systems, which include Ubuntu::
+Linux compilers from the system
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    sudo apt-get install build-essential python3-dev python3-setuptools \
-                     python3-pip
-    
-and then::
+Installing scikit-learn from source without using conda requires you to have
+installed the scikit-learn Python development headers and a working C/C++
+compiler with OpenMP support (typically the GCC toolchain).
 
-    pip3 install numpy scipy cython
+Install build dependencies for Debian-based operating systems, which include
```

Comment:
```suggestion
Install build dependencies for Debian-based operating systems, e.g.
```

(not sure whether "include" takes an s here?)

Reply:
"operating systems" is plural so no "s". But fair enough.